src/test/kotlin/no/nav/helsearbeidsgiver/altinn/buildAltinnClient.kt

```diff
@@ -1,6 +1,6 @@
 package no.nav.helsearbeidsgiver.altinn
 
-import io.ktor.http.*
+import io.ktor.http.HttpStatusCode
```

Comment:
Jeg foretrekker også spesifikke importer. Er det konfigurert i prosjektet slik at intellij automatisk bruker spesifikk klasser?

Reply:
Kan man ha kode i prosjektet som overrider Intellij sine import-settings? Så vidt jeg vet må endre Intellij-innstillingene sine selv (bare én gang, ikke for hvert prosjekt).

Ktlint er derimot satt opp til å tolerere wildcard imports, noe jeg med glede fjerner.